spec/seeds/peeps_seeds.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS peeps;
+
+-- Table Definition
+CREATE TABLE peeps (
+  id SERIAL PRIMARY KEY,
+  content text,
+  time time,
```

Comment:
What about using timestamp? It can store both date and time. Because Ruby has the function called Time.now that also includes the timezone, it might be interesting to add this as well. You can always decide what information you actually want to display in the app.